recipe/meta.yaml

```diff
@@ -13,15 +13,16 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 5
 
 requirements:
   build:
     - {{ compiler('c') }}
     - cmake
   host:
     - python
-    - {{ pin_compatible('numpy') }}
+    - openblas  # [Linux]
```

Comment:
You can force openblas numpy by adding, `blas 1.1 openblas  # [unix]` to host and run requirements.

Reply:
OK. Done.